src/mscorlib/corefx/System/Globalization/CultureData.Unix.cs

```diff
@@ -106,10 +83,41 @@ private unsafe bool InitCultureData()
                     this.sName = this.sWindowsName.Substring(0, index);
                 }
             }
+            return true;
+        }
 
+        internal static bool GetLocaleName(string localeName, out string windowsName)
+        {
+            // Get the locale name from ICU
+            StringBuilder sb = StringBuilderCache.Acquire(ICU_ULOC_FULLNAME_CAPACITY);
+            if (!Interop.GlobalizationInterop.GetLocaleName(localeName, sb, sb.Capacity))
+            {
+                StringBuilderCache.Release(sb);
+                windowsName = null;
+                return false; // fail
+            }
+
+            // Success - use the locale name returned which may be different than realNameBuffer (casing)
+            windowsName = StringBuilderCache.GetStringAndRelease(sb); // the name passed to subsequent ICU calls
+            return true;
+        }
+
+        internal static bool GetDefaultLocaleName(out string windowsName)
+        {
+            // Get the locale name from ICU (pass null for default)
```

Comment:
We aren't technically passing null here right?  Is that okay?  If so, can we clean up the comment.